### DISCUSSION TITLE
Assistant Apprentice mind swipe

### DISCUSSION THREAD

**Discussion by Ivanzyfer** [FakeFishGames/Barotrauma#15291] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When resetting talents with mind swipe, the talents received from Graduation Ceremony remain. With this, you can get all the talents for one assistant

### Reproduction steps

1. Start campaign
2. Take the talent Graduation Ceremony
3. Use mind wipe
4. Reset the talents
5. Take the talent Graduation Ceremony again
6. Repeat for more extra talent
![image](https://github.com/user-attachments/assets/56ca9b2b-7060-419a-b790-023a1d392571)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14858
